.travis.yml

```diff
@@ -5,7 +5,7 @@ script:
   # make path to locally store JWT secret
   - mkdir .secret
   - bash .scripts/buildDB.sh
-  - mvn clean package jetty:run &
+  - mvn clean spotbugs:spotbugs package jetty:run &
```

Comment:
Maybe we should move spotbugs after `package`?

Reply:
Outdated. In fact, already moved.,